submissions/DawoduRahmon/htmlForms/contact.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<head>
+	<meta charset="UTF-8">
+    <meta name="viewport" content="width=div, initial-scale=1.0">
+    <title>Contact-Form</title>
+</head>
+<body>
```

Comment:
You did not close this
@Dawodurahmonn

Reply:
Do you mean the body tag ma?

Reply:
Yes @Dawodurahmonn . You did not close the body tag.